File: ObligatorioDA2/Models/PuntoTuristicoModel.cs
using Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Obligatorio.Models
{
    public class PuntoTuristicoModel
    {
        public string Nombre { get; set; }
        public string Descripcion { get; set; }
        public Imagen Imagen { get; set; }
        public List<string> Categorias { get; set; } 
    }
}

File: SPA/FrontEnd/src/app/Models/Punto/PuntoTuristicoModel/punto-turistico-model.ts
import { Imagen } from '../../Imagen/imagen';

export class PuntoTuristicoModel {
    nombre: string;
    descripcion: string;
    imagen: Imagen;
    categorias: string[];

    constructor(name: string, desc: string, imagen: Imagen, categorias: string[]) {
        this.nombre = name;
        this.descripcion = desc;
        this.imagen = imagen;
        this.categorias = categorias;
    }
}
